src/encoding/lookup.py

```diff
@@ -6,7 +6,7 @@ class LookupTable:
     rows: Set[int]
     random: int
 
-    def __init__(self, columns: Sequence[str], random: int = 123) -> None:
+    def __init__(self, columns: Sequence[str], random: int = 5566) -> None:
```

Comment:
The small value causes a collision in the compression.

For example, these two rows have the same commitment.

- a = 123, b = 0, compression = 123 * 1 + 0 * 123**1
- a = 0, b = 1, compression = 0 * 1 + 1 * 123**1

Reply:
I think if i is derieved from x8s then it should be impossible to have a collision using the values in the table.